src/components/organisms/LinkCard.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropType from 'prop-types';
+import './LinkCard.scss';
+
+const LinkCard = ({ story }) => (
+  <article className="card">
+    <div className="side-bar">
+      <div className="side-bar-icon">icon</div>
+      <div className="side-bar-links">
+        <p>links</p>
+        <p>stars</p>
+      </div>
+    </div>
+    <div className="card-info">
+      <header>
+        <span className="byline">
+          {`${story.authorName} on ${new Date(story.createdDate * 1000).toLocaleDateString('en-US', { year: 'numeric', month: 'long', day: '2-digit' })}`}
```

Comment:
the Timestamp object actually has a `toDate()` method you can call. 
```suggestion
          {`${story.authorName} on ${story.createdDate?.toDate().toLocaleDateString('en-US', { year: 'numeric', month: 'long', day: '2-digit' })}`}
```

I added the optional chaining on createdDate for now just because currently the dummy data has no createdDate information. And if you haven't before, you can add a createdDate field to one or both of the records in the firestore emulator to confirm that this line of code works.